openapi.yaml

```
@@ -2165,6 +2177,631 @@ paths:
+              properties:
+                url:
+                  type: string
+                  format: binary
+                fileName:
+                  type: string
+                  format: binary
+        required: true
+      responses:
+        405:
+          description: Invalid input
+          content: {}
+      security:
+        - oscal_auth:
+            - write:poams
+            - read:poams
+      x-codegen-request-body-name: body
+  /plan-of-action-and-milestones/{poamId}:
+    get:
+      tags:
+        - OSCAL Plan of Action and Milestone
+      summary: Finds an OSCAL plan of action and milestone by ID
+      description: Returns a single OSCAL Plan of Action and Milestone
+      operationId: getPoamById
+      parameters:
+        - name: poamId
+          in: path
+          description: ID of plan of action and milestones to return
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/OSCALPoam"
+        400:
+          description: Invalid ID supplied
+          content: {}
+        404:
+          description: Plan of Action and Milestones not found
+          content: {}
+      security:
+        - oscal_auth:
+            - read:poams
+    patch:
+      tags:
+        - OSCAL Plan of Action and Milestone
+      summary: Updates an existing OSCAL plan of action and milestone
+      operationId: updatePoam
+      parameters:
+        - name: poamId
+          in: path
+          description: ID of plan of action and milestone to return
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: Plan of action and milestone object to be updated
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/OSCALPoamUpdateExample"
+        required: true
+      responses:
+        400:
+          description: Invalid ID supplied
+          content: {}
+        404:
+          description: Plan of action and milestones not found
+          content: {}
+        405:
+          description: Validation exception
+          content: {}
+        409:
+          description: Conflict with current state
+      security:
+        - oscal_auth:
+            - write:poams
+            - read:poams
+      x-codegen-request-body-name: body
+    put:
+      tags:
+        - OSCAL Plan of Action and Milestone
+      summary: Replaces an existing OSCAL plan of action and milestone
+      operationId: replacePoam
+      parameters:
+        - name: poamId
+          in: path
+          description: ID of plan of action and milestone to replace.
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: Plan of action and milestone object to be replaced.
```

```suggestion
          description: ID of plan of action and milestone to replace
          required: true
          schema:
            type: string
      requestBody:
        description: Plan of action and milestone object to be replaced
```

For consistency purposes, we should delete any periods in the descriptions. There are a couple other places where this occurs

---

I believe prior descriptions had periods in them. Should we keep or remove periods in descriptions?

---

Probably best to remove those too, even though they aren't part of this issue.